src/lib.rs

```diff
@@ -3,8 +3,6 @@
 
 #![warn(missing_docs)]
 #![warn(unreachable_pub)]
```

Comment:
I know this is not part of your code, but can we remove this now that we have removed some of the features?